src/app/app-routing.module.ts

```diff
@@ -9,13 +9,27 @@ import { ConfirmationComponent } from './components/content/confirmation/confirm
 import { AboutComponent } from './components/content/about/about.component';
 import { FaqComponent } from './components/content/faq/faq.component';
 import { NotFoundComponent } from './components/content/not-found/not-found.component';
+import { AdminComponent } from './components/content/admin/admin.component';
+import { DashboardComponent } from './components/content/admin/dashboard/dashboard.component';
+import { StatisticsComponent } from './components/content/admin/statistics/statistics.component';
+import { MembersComponent } from './components/content/admin/members/members.component';
+import { EditRapportComponent } from './components/content/admin/edit-rapport/edit-rapport.component';
+import { AuthenticationGuard } from './guards/authentication.guard';
+import { AddPartnerComponent } from './components/content/admin/add-partner/add-partner.component';
 
-const routes: Routes = [
+export const routes: Routes = [
```

Comment:
why are you exporting the routes used for the actual product? if you're exporting them to be used in tests, it is better to mock routes with the `RoutingTestingModule.withRoutes(<MockRoutesObject>)` while testing then using the actual routes. this reduces dependency on things that could change in the future.

Reply:
done